package.json

```diff
@@ -112,12 +112,14 @@
     "@polymer/paper-tooltip": "^3.0.1",
     "@polymer/polymer": "^3.4.1",
     "@tensorflow/tfjs": "^2.3.0",
+    "@types/resize-observer-browser": "^0.1.3",
```

Comment:
move this to devDependency. type files are used for building the tool and is not to be used in the runtime (I know that for non-Node projects, it is kinda awkward and arbitrary distinction but devDep is consistent with other type imports)

Reply:
Done.